src/main/kotlin/eu/kevin/api/models/general/projectSettings/GetProjectSettingsResponse.kt

```diff
@@ -7,7 +7,7 @@ import kotlinx.serialization.Serializable
 data class GetProjectSettingsResponse(
     val scopes: List<String>,
     val isSandbox: Boolean,
-    val paymentMethods: List<String>,
+    val paymentMethods: List<String>? = null,
```

Comment:
If `null`, default value is `bank`. Reference: https://gokevin.atlassian.net/browse/DSB-113